tests/ui/feature-gates/feature-gate-const-refs-to-static.stderr

```diff
@@ -0,0 +1,28 @@
+error[E0013]: constants cannot refer to statics
+  --> $DIR/feature-gate-const-refs-to-static.rs:4:19
+   |
+LL | const C1: &i32 = &S;
+   |                   ^
+   |
+   = help: consider extracting the value of the `static` to a `const`, and referring to that
```

Comment:
You've probably already planned on adding that but Imma comment anyway:

```diff
+    = note: see issue #NNNNNN <https://github.com/rust-lang/rust/issues/NNNNNN> for more information
```

And for `tcx.sess.is_nightly_build()`:

```diff
+ help: add `#![feature(const_refs_to_static)]` to the crate attributes to enable
```

Reply:
That seems like an issue with our const stability checks -- not related to this PR.

Reply:
Ah I have to put the feature gate into `build_error` as well.